src/index.ts

```diff
@@ -28,6 +25,18 @@ export default function generateSitemap(options: UserOptions = {}) {
   writeRobotFile(getResolvedPath('robots.txt', resolvedOptions), resolvedOptions)
 }
 
+export function getRoutes(options: ResolvedOptions) {
+  return [
+    ...glob.sync('**/*.html', { cwd: options.outDir }).map((route) => {
+      const parsedRoute = parse(route.replace(/index\.html/g, ''))
+      return (
+        join('/', parsedRoute.dir, parsedRoute.name)
+      )
+    }),
+    ...options.dynamicRoutes.map(route => join('/', join(parse(route).dir, parse(route).name))),
```

Comment:
Your fix needed to be on dynamic routes too

Reply:
Oppps, sorry, missed that part! Thanks!